doc/howtos/rdtraining/13_inheritance.rst

```diff
@@ -0,0 +1,244 @@
+root views, and can add or remove content from their parent.
+
+An extension view references its parent using the ``inherit_id`` field, and
+instead of a single view its ``arch`` field is composed of any number of
+``xpath`` elements selecting and altering the content of their parent view:
+
+.. code-block:: xml
+
+    <record id="inherited_model_view_form" model="ir.ui.view">
+        <field name="name">inherited.model.form.inherit.test</field>
+        <field name="model">inherited.model</field>
+        <field name="inherit_id" ref="inherited.inherited_model_view_form"/>
+        <field name="arch" type="xml">
+            <!-- find field description and add the field
+                 new_field after it -->
+            <xpath expr="//field[@name='description']" position="after">
+              <field name="new_field"/>
+            </xpath>
+        </field>
+    </record>
+
+``expr``
+    An XPath_ expression selecting a single element in the parent view.
+    Raises an error if it matches no element or more than one
+``position``
+    Operation to apply to the matched element:
+
+    ``inside``
+        appends ``xpath``'s body at the end of the matched element
+    ``replace``
+        replaces the matched element with the ``xpath``'s body, replacing any ``$0`` node occurrence
+        in the new body with the original element
+    ``before``
+        inserts the ``xpath``'s body as a sibling before the matched element
+    ``after``
+        inserts the ``xpaths``'s body as a sibling after the matched element
+    ``attributes``
+        alters the attributes of the matched element using special
+        ``attribute`` elements in the ``xpath``'s body
+
+When matching a single element, the ``position`` attribute can be set directly
+on the element to be found. Both inheritances below will give the same result.
+
+.. code-block:: xml
+
+    <xpath expr="//field[@name='description']" position="after">
+        <field name="idea_ids" />
+    </xpath>
+
+    <field name="description" position="after">
+        <field name="idea_ids" />
+    </field>
+
+The view part inheritance from the previous practical example can be found
+`here <https://github.com/odoo/odoo/blob/691d1f087040f1ec7066e485d19ce3662dfc6501/addons/account_fleet/views/account_move_views.xml#L3-L17>`__.
+
+.. exercise:: Add fields on the Users view
+
+    Add the ``property_ids`` field on the ``base.view_users_form`` in a new page of the notebook.
+
+    Tip: an example of inheritance of the users' view can be found
+    `here <https://github.com/odoo/odoo/blob/691d1f087040f1ec7066e485d19ce3662dfc6501/addons/gamification/views/res_users_views.xml#L5-L14>`__. 
+
+Inheritance is extensively used in Odoo due to its modular conception. Do not hesitate to read
+the corresponding documentation for more info!
+
+In the :ref:`next chapter <howto/rdtraining/14_other_module>`, we will learn how to interact with
+other modules.
+
+.. _XPath: http://w3.org/TR/xpath
```

Comment:
Maybe link to somethign a bit less brutal than the W3C spec? They're not known for their readability...

Reply:
e.g. http://zvon.org/comp/r/tut-XPath_1.html#Pages~List_of_XPaths